internal/service/service.go

```diff
@@ -50,14 +51,15 @@ func (e *Service) DeployService(ctx *context.Context, request *serviceProto.Depl
 			message = response.ServiceResponse.Message
 			message += fmt.Sprintf("\n Service %s %s", response.ServiceResponse.ServiceStatus.ServiceAction, response.ServiceResponse.ServiceStatus)
 			for _, compMessage := range response.ServiceResponse.ComponentsStatus {
-				message += fmt.Sprintf("\n Component %s %s %s", compMessage.ComponentName, compMessage.ComponentAction, compMessage.ComponentStatus)
+				message += fmt.Sprintf("\n Component %s %s %s %s", compMessage.ComponentName, compMessage.ComponentAction, compMessage.ComponentStatus,compMessage.Error)
 			}
 			spinnerInstance.Prefix = fmt.Sprintf(" %s  ", message)
+			prevMessage = message
 			spinnerInstance.Start()
 		}
 	}
 
-	log.Info(message)
+	log.Info(prevMessage)
```

Comment:
do we need this?

Reply:
yes, this is required to print the last message because it is cleared before printing